Homework 1

IvanFed3 opened the pull request:
And some files from started homework 2 (not completed)

IvanFed3 committed:
Homework1

IvanFed3 committed:
Add some new element

For update to Homework 2  need to marge this version.

shorodilov commented on py4dj/settings.py:
```diff
@@ -0,0 +1,124 @@
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'task',
```
Відділяй власні застосунки порожнім рядком.
Це гарна практика.

shorodilov commented on task/templates/register.html:
```diff
@@ -0,0 +1,10 @@
+    <title>Title</title>
+</head>
+<body>
+
```
Тут не вистачає форми реєстрації

shorodilov commented on task/templates/login.html:
```diff
@@ -0,0 +1,10 @@
+    <title>Title</title>
+</head>
+<body>
+
```
Тут не вистачає форми логіна.

shorodilov reviewed (approved):
Ага, я зрозумів - ти намішав в цей PR частину наступної домашки.
Поки пробачу, але будь ласка, будь уважним. Твої дії повині бути усвідомленими.

IvanFed3 committed:
Homework 1 and Homework 2

Тут завершена перша та друга робота. Я вибачаюсь за кількість файлів. В домашній роботі по блогу та моделі виправлюсь.
